template/sqstat_html.tpl

```diff
@@ -122,11 +118,11 @@ var now = new Date();
 
 var year = now.getFullYear();
 var month = now.getMonth();
-//console.log(month)
+console.log(month)

```

Comment:
This should be commented out otherwise it spams the browser console

Reply:
Yes, these were in the original PR that were never merged and what he was trying to resolve with his branch conflicts.  I told him to put them both in this one

Reply:
Sounds good. Then I think all it needs is commenting out that debug line.